namui/src/namui/render/rendering_tree/visit.rs

```diff
@@ -350,84 +347,173 @@ mod tests {
                         }
                         "2" => {
                             assert_approx_eq!(f32, local_xy.x, 10.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, 10.0, ulps = 2);
+                            call_count += 1;
                         }
                         "3" => {
                             assert_approx_eq!(f32, local_xy.x, -90.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -190.0, ulps = 2);
+                            call_count += 1;
                         }
                         "4" => {
                             assert_approx_eq!(f32, local_xy.x, -110.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -220.0, ulps = 2);
+                            call_count += 1;
                         }
                         "5" => {
                             assert_approx_eq!(f32, local_xy.x, -10.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -10.0, ulps = 2);
+                            call_count += 1;
                         }
                         "6" => {
                             assert_approx_eq!(f32, local_xy.x, -90.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -90.0, ulps = 2);
+                            call_count += 1;
                         }
                         "7" => {
                             assert_approx_eq!(f32, local_xy.x, -30.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -30.0, ulps = 2);
+                            call_count += 1;
                         }
                         "8" => {
                             assert_approx_eq!(f32, local_xy.x, -110.0, ulps = 2);
                             assert_approx_eq!(f32, local_xy.y, -110.0, ulps = 2);
+                            call_count += 1;
                         }
                         "9" => {
-                            assert_approx_eq!(f32, local_xy.x, -180.0, ulps = 2);
-                            assert_approx_eq!(f32, local_xy.y, -380.0, ulps = 2);
+                            assert_approx_eq!(f32, local_xy.x, -45.0, ulps = 2);
+                            assert_approx_eq!(f32, local_xy.y, -95.0, ulps = 2);
+                            call_count += 1;
                         }
                         "10" => {
-                            assert_approx_eq!(f32, local_xy.x, -200.0, ulps = 2);
-                            assert_approx_eq!(f32, local_xy.y, -400.0, ulps = 2);
+                            assert_approx_eq!(f32, local_xy.x, -65.0, ulps = 2);
+                            assert_approx_eq!(f32, local_xy.y, -115.0, ulps = 2);
+                            call_count += 1;
+                        }
+                        _ => {}
+                    }
+                }
+            }
+            ControlFlow::Continue(())
+        });
+        assert_eq!(call_count, 11);
```

Comment:
Actually the reason why the test on previous PR was passed is, I didn't check 9 and 10. So I made it sure that call_count is 11, so 9 and 10 is called.